doc/api/n-api.md

```diff
@@ -3684,6 +3684,23 @@ NAPI_EXTERN napi_status napi_run_script(napi_env env,
 - `[in] script`: A JavaScript string containing the script to execute.
 - `[out] result`: The value resulting from having executed the script.
 
+## libuv event loop
+
+N-API provides a function for getting the current event loop associated with
+a specific `napi_env`.
+
+### napi_uv_get_event_loop
```

Comment:
This should be `napi_get_uv_event_loop` I think.

Reply:
@cjihrig Thanks, fixed!